### DISCUSSION TITLE
Gaganode shutdown

### DISCUSSION THREAD

**Discussion by superit63** [engageub/InternetIncome#121] [Q&A]
I am running gaganode with 10 proxies but when I checked gaga dashboard, some of node has been "DOWN". After I checked logs inside the container, I see that there is some issues on network (like proxy speed, latency...) or environment is bad ==> Node shutdown although container is still alive.
I have restarted the docker containers that it works fine.
Anyone how to fix it automatically instead of looking inside each container and restart one by one!!!

Thanks a lot

**Comment by engageub** [comment]
Hello,
Other users have also reported such errors in the [discord group](https://discord.gg/z6YfSHDkmS) already. 
Here is the response given.
```
This error is mainly because of a poor/unstable internet connection. Please check your internet connection

```

![image](https://github.com/engageub/InternetIncome/assets/103608400/0751d2e1-9f52-47fc-86a7-5a01b622bee3)


I would also like to know if you made any configuration changes related to proxies apart from the default values.


If you wish to make better earnings, please follow the wiki mentioned below.
https://github.com/engageub/InternetIncome/wiki/Maintaining-Optimal-IP-Coefficient-for-Gaganode


Thank you

**Reply by superit63** [reply]
Hi Mr/Ms,

I am appreciate your code, your answer. Its clear now.
I dont made any configuration changes related to proxies apart from the default values. So, the error is just in my proxy (poor internet connection) and we dont have solution as restart docker to retry it?

Btw, do you have any community in telegram/discord? I would like to join. Thanks

**Reply by engageub** [reply]
Hello,
Thank you for the confirmation. 
Yes, the error is related to your proxy or internet connection according to the discord statement mentioned.
However, you can mention any other errors in the discord group.
It should happen to others as well who are using it but not all have such issues. So it might be your proxy or internet connection as mentioned. 
May be DNS is blocking some websites which gaganode requires. 
I have not created any group in telegram/discord as github already has Discussion thread. 
If there is anything you may want to ask, you can post here. 

Thank you
